react-hooks-counter/src/counter.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+
+const Counter = () => {
+  const [text, setText] = useState('Button has not been clicked');
```

Comment:
Instead of text here, try setting a boolean state, then conditionally choose the text.

```js
const [isClicked, setIsClicked] = useState(false);

const text = isClicked
  ? 'The button has been clicked'
  : 'Button has not been clicked';
```